### ISSUE TITLE
A/UX

### ISSUE THREAD

**Issue by bugaevc** [mihaip/infinite-mac#129]
I'd love to run A/UX through this! [Unlike NeXTSTEP](https://github.com/mihaip/infinite-mac/issues/121), this is actual Mac OS.

**Comment by mihaip** [comment]
Porting https://github.com/pruten/shoebill and/or QEMU (see #72) would be required for A/UX.

**Comment by adespoton** [comment]
I was thinking about how Basilisk II does what it does by patching out the rom, and Shoebill does what it does... by patching out the rom.  If Basilisk II is compiled in 24-bit mode, it's essentially emulating a Mac II, (almost) like Shoebill does.  So I'm going to investigate how much work is involved in a) adding the extra emulated hardware needed to BII, and b) adding a conditional patch that would use pruten's bootloader instead of the regular Mac OS bootloader.

My thought was this: the two emulators have different components missing, but fundamentally function in the same manner.  So if I (or someone else with more time) can implement the pieces in Shoebill that are missing from BII in BII, then we have an all-round better emulator with more capabilities.  The sticking point here might be SCSI, as Shoebill emulates it but BII intentionally patches out the drive hardware handling code completely.  However, since we're not using the actual drive handling code for anything, it might be enough to present a SCSI-looking device to A/UX and then just route it through the regular patch routine BII uses.

**Comment by dingusdev** [comment]
A/UX is now supported by Snow. Shoebill has not been touched for a while.

**Comment by mihaip** [comment]
Yes, closing in favor of #437 (https://infinitemac.org/run?disk=A%2FUX+3.1.1)
